tests/api_tests/base/test_serializers.py

```diff
@@ -50,3 +53,46 @@ def test_invalid_param_raises_bad_request(self):
 
         res = self.app.get(self.url, params={'related_counts': 'fish'}, expect_errors=True)
         assert_equal(res.status_code, http.BAD_REQUEST)
+
+
+class TestJSONAPIHyperlinkedIdentityField(DbTestCase):
```

Comment:
@sloria can you explain the motivation for wanting to use `DBTestCase` over `APITestCase` (and the need to the `make_drf_request` util?